lib/krane/cli/deploy_command.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Krane
+  module CLI
+    class DeployCommand
+      DEFAULT_DEPLOY_TIMEOUT = '300s'
+      PROTECTED_NAMESPACES = %w(
+        default
+        kube-system
+        kube-public
+      )
+      OPTIONS = {
+        "bindings" => { type: :array, banner: "foo=bar,abc=def",
+                        desc: "Expose additional variables to ERB templates (format: k1=v1,k2=v2, JSON string or file "\
+                          "(JSON or YAML) path prefixed by '@')" },
+        "filename" => { type: :string, banner: '/tmp/my-resource.yml', aliases: :f,
+                        desc: "Path to a file that contains the configuration to apply" },
+        "global-timeout" => { type: :string, banner: "duration", default: DEFAULT_DEPLOY_TIMEOUT,
+                              desc: "Max duration to monitor workloads correctly restarted" },
+        "protected-namespaces" => { type: :string, banner: "list,of,namespaces",
```

Comment:
Curious what you think of this form instead of how it was before?

Reply:
I think it's a good idea. It might be useful to be able to add other protected namespaces to the list besides the default ones as well. I don't really see much downside in letting people change this.

Reply:
Why not have this as type `:array`?

Reply:
Maybe I'm using it wrong, but it seems that Thor arguments of type `:array` don't allow for empty values, raising `Thor::MalformattedArgumentError: No value provided for option '--protected-namespaces'`. I didn't find any obvious workaround for this :( I will change it to array though, as it would make more sense semantically.